plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/plan/CodeModernizerPlanEditor.kt

```diff
@@ -3,178 +3,495 @@
+                        }
+                    add(stepsIntroSubtitle)
+                    border = CodeModernizerUIConstants.STEPS_INTRO_BORDER
+                }
+            add(stepsIntro, CodeModernizerUIConstants.transformationPlanPlaneConstraint)
+            // ignore step 0 since that contains info used to create tables in the plan
+            plan.transformationSteps().drop(1).forEachIndexed { _, step ->
+                var row: JPanel? = null
+                row = transformationStepPanel(step)
+                add(row, CodeModernizerUIConstants.transformationPlanPlaneConstraint)
+            }
+            border = CodeModernizerUIConstants.TRANSFORMATION_PLAN_PANEL_BORDER
+        }
+
+    private fun transformationPlanAppendix(table: PlanTable): JPanel {
+        val panel =
+            JPanel(GridBagLayout()).apply {
+                val constraints = CodeModernizerUIConstants.transformationPlanAppendixConstraint
+                val appendixTitle =
+                    JLabel(message("codemodernizer.migration_plan.body.info.appendix_title")).apply {
+                        font =
+                            font.deriveFont(
+                                CodeModernizerUIConstants.FONT_CONSTRAINTS.BOLD,
+                                CodeModernizerUIConstants.PLAN_CONSTRAINTS.SUBTITLE_FONT_SIZE,
+                            )
+                        border = CodeModernizerUIConstants.APPENDIX_BORDER
+                    }
+                add(appendixTitle, constraints)
+
+                val tableName =
+                    JLabel(table.name).apply {
+                        font =
+                            font.deriveFont(
+                                CodeModernizerUIConstants.FONT_CONSTRAINTS.PLAIN,
+                                CodeModernizerUIConstants.PLAN_CONSTRAINTS.TABLE_NAME_FONT_SIZE,
+                            )
+                        border = CodeModernizerUIConstants.TABLE_NAME_BORDER
+                    }
+                add(tableName, constraints)
+
+                val renderedAppendixTable =
+                    createTable(table).apply {
+                        tableHeader.reorderingAllowed = false // so that file click detection works; see below
+                    }
+
+                // make file paths blue
+                val cellRenderer =
+                    object : DefaultTableCellRenderer() {
+                        override fun getTableCellRendererComponent(
+                            table: JTable?,
+                            value: Any?,
+                            isSelected: Boolean,
+                            hasFocus: Boolean,
+                            row: Int,
+                            column: Int,
+                        ): JLabel {
+                            val label = super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column) as JLabel
+                            label.foreground = Color(89, 157, 246)
```

Comment:
does this color need to be theme specific?

Reply:
No, this is just a shade of blue for our hyperlinks.